xapian-applications/omega/templates/query

```diff
@@ -37,87 +37,96 @@ $if{$opt{topterms},$include{inc/toptermsjs}}
-<hr>Searching $nice{$dbsize} documents
-},
+Searching $nice{$dbsize} documents
+}</p>,
 $if{$not{$msizeexact},
  $nice{$add{$topdoc,1}}-$nice{$last} of about $nice{$msize} matches,
  $if{$and{$eq{$last,$msize},$eq{$topdoc,0}},
   All $nice{$msize} matches,
   $nice{$add{$topdoc,1}}$if{$ne{$add{$topdoc,1},$last},-$nice{$last}} of exactly $nice{$msize} matches}
 }
-<hr>
-</center>
-$list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;$nice{$freq{$_}}},Term frequencies: ,$. ,}
-<br><small>Search took $time seconds</small>
+<br/>
+$list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},,,}:&nbsp;$nice{$freq{$_}}},Term frequencies: ,$. ,}
+Search took $time seconds
+</p>
+${Results}
 <table>
-$hitlist{<tr><td valign=top>
+$hitlist{<tr><td class="result-metadata">
 ${<img src="/icons/omega/score-$div{$percentage,10}.png" alt="$percentage%" height=16 width=32>}
-<div title="$percentage%" style='float:left;width:60px;height:10px;border:1px solid black;margin-top:4px;'>
-<div style='width:$div{$mul{$percentage,6},10}px; height:10px; background-color: red;'>
+<div class="result-metadata-progress-container" title="$percentage%" >
+<div class="result-metadata-progress" style='width:$div{$mul{$percentage,6},10}px;'>
 </div></div>
-<div style='float:left;margin-top:2px;font-size:x-small;'>
+<div class="result-metadata">
 $if{$and{$field{modtime},$ne{$field{modtime},-1}},
 <script type="text/javascript">write_date($add{$field{modtime}});</script>
 <noscript>
-<span title="$html{$date{$field{modtime},%Y-%m-%d %H:%M:%S}}">
-Modified:<br><b>$html{$date{$field{modtime},%Y-%m-%d}}</b></span>
-</noscript><br>
+<span>Modified:<br/><time datetime="$html{$date{$field{modtime},%Y-%m-%d %H:%M:%S}}">
+$html{$date{$field{modtime},%Y-%m-%d}}</time></span>
+</noscript>
 }
-$if{$field{language},Language: <b>$html{$field{language}}</b><br>}
-$if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesize{$field{size}}}</b></span><br>}
-<div title="Mark this document as relevant" style="float:left;background-color:#cfc;border:1px solid green;margin-top:2px;">
+$if{$field{language},Language: $html{$field{language}}}
+$if{$field{size},<br/><span title="$html{$field{size}} bytes">Size:$html{$filesize{$field{size}}}</span>}
+<div title="Mark this document as relevant">
 <input type=checkbox name=R value=$id$if{$relevant, checked}>
 </div>
 </div>
 </td>
-<td><b><a href="$html{$field{url}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
-<small>$highlight{$snippet{$field{sample}},$terms}$if{$field{sample},...}</small><br>
-<a href="$html{$field{url}}">$html{$field{url}}</a><br>
-<small>
+<td class="result-data"><header class="result-data"><a href="$html{$field{url}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></header>
+<span class="result-data" style="display: block;">
+$highlight{$snippet{$field{sample}},$terms}$if{$field{sample},...}</span>
```

Comment:
You can make `blockquote` look however you like. It's also not inline content (although you get away with it because it follows a `header`, which will probably be default `display: block`), so it would be `div` not `span` even if you don't use `blockquote`; the fact that you're explicitly setting those `.result-data` elements to `display: block;` kind of underscores that.

And if you're using `div`, you should use `blockquote`, because:

> [The blockquote element represents a section that is quoted from another source.](https://w3c.github.io/html/grouping-content.html#the-blockquote-element)

It's common in "real" web systems, with more complex CSS, to start with a CSS reset or normalisation, which is a set of rules that turns everything pretty much flat and boring, but provides consistency across browsers (since, for instance, the indent used by browsers for `blockquote` doesn't have to be standard across all of them). In this case, `blockquote` would look identical to `div` anyway.